package.json

```diff
@@ -94,7 +94,8 @@
 		"source-registry-storage-adapter": "github:devx-services/source-registry-storage-adapter#main",
 		"util": "^0.12.5",
 		"uuid": "^8.3.2",
-		"yaml": "^2.4.2"
+		"yaml": "^2.4.2",
+		"newrelic": "^11.0.0"
```

Comment:
From what I understand, `newrelic` isn't actually a dependency of the CLI plugin. However, since plugin-hooks uses `newrelic` alongside MGQL, we have to add it here?

Reply:
Yes, unfortunately due to our setup we have to add newrelic even thought its not being used on the CLI.

Maybe in the future we can refactor it to not need newrelic while not breaking the hooks package.